src/Components/ProductListingPage/ProductsSection/ProductsCard/ProductsCard.jsx

```diff
@@ -57,10 +65,18 @@ const ProductsCard = ({ item }) => {
               </button>
               <div className="card-nav-icon">
                 <button
-                  onClick={addWishlistClick}
+                  onClick={
+                    wishlist.includes(item)
+                      ? removeFromWishlist
+                      : addWishlistClick
```

Comment:
Instead of attaching clickListeners like this. Attach a function here, which takes the ***item*** as argument and check the ```wishlist.includes(item)``` logic inside the function.
something like
```
const handleWishListClick=(item)=> {
   if(wishlist.includes(item)){
       removeFromWishlist()
   } else {
       addWishlistClick()
   }
}

```

Reply:
Noted.